test/JDBC/expected/BABEL-GRANT.out

```diff
@@ -174,7 +174,7 @@ GO
 REVOKE SELECT ON SCHEMA::scm FROM guest;
 GO
 
-GRANT SHOWPLAN ON OBJECT::t1 TO guest;  -- unsupported permission
+GRANT showplan ON OBJECT::t1 TO guest;  -- unsupported permission
```

Comment:
Why do we need this change? Only the letter casing is changed here

Reply:
Whether the input "SHOWPLAN" is in lower/upper case, the error message should have that KEYWORD in upper case always.